lib/resetState.ts

```diff
@@ -1,3 +1,9 @@
+type SerializableObjVal = SerializableObj | string | number;
+
+type SerializableObj = {
+  [key in string]: SerializableObjVal;
```

Comment:
Can the array be the `SerializableObjVal`?

Reply:
Updated typings ;) still, this naive impl using `JSON.parse(JSON.stringify(...)` won't cope with `{a: undefined}` (it will transform it into `{}`) I'm thinking of implementing more sophisticated (comparing to `JSON.parse`) method which recursively clones object

Reply:
Things are getting complicated with implementing your own `JSON.parse`. It would be sufficient for the yet another open-source project probably xD